Kinvey/Kinvey/ValidationStrategy.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  DataValidationStrategy.swift
+//  Kinvey
+//
+//  Created by Victor Hugo Carvalho Barros on 2017-10-12.
+//  Copyright © 2017 Kinvey. All rights reserved.
+//
+
+import Foundation
+
+public enum ValidationStrategy {
+    
+    /// Percentagem between 0.0 and 1.0
```

Comment:
Typo.
Also, it'll help to define this better. I suggest saying -
"Percentage between 0.0 and 1.0. This number specifies the fraction of entities in a backend response that are validated. Validating a higher percentage of entities upfront results in a performance penalty."